lib/p2p/Pool.ts

```diff
@@ -19,8 +19,9 @@ type PoolConfig = {
 
 /** A pool of peers for handling all network activity */
 class Pool extends EventEmitter {
+  public  peers: PeerList = new PeerList();
```

Comment:
It seems like the only thing we're using `peers` for outside this class is to get the number of keys aka length. In that case, I think it's better to leave this private and add a public method here to return this info. Also, if we're only concerned with the number of *connected* peers, which for getInfo purposes I think makes sense, we should count only the peers that are currently connected.

Reply:
Isn't every peer in `peers` connected? I thought so because in `broadcastOrder` the order gets send to every peer in `peers` and sending a package to peers that aren't connected would make no sense.